### DISCUSSION TITLE
My minimum iOS configuration is iOS 13, but  after  pod install lotties success,    still compile errors

### DISCUSSION THREAD

**Discussion by ywz364kf** [airbnb/lottie-ios#2395] [General]
![image](https://github.com/airbnb/lottie-ios/assets/6925054/a3290234-beaa-4c73-98f2-ae39cc76c2a7)

pod 'lottie-ios', '4.4.3'

after pod instlal success, compile error

**Comment by aakashbondwal** [comment]
Works fine for me. Check you simulator iOS version as well

**Comment by ifamirhasan** [comment]
I have same issue. after every pod install i have to set minimum ios version in lottie-ios project to ios 13

**Comment by calda** [comment]
The pod spec lists iOS 13 as the minimum version, is this configured incorrectly? https://github.com/airbnb/lottie-ios/blob/master/lottie-ios.podspec#L26
